L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -517,30 +516,41 @@ std::vector<std::pair<GlobalPoint, double>> HGCalHistoSeedingImpl::computeSecond
 
 void HGCalHistoSeedingImpl::findHistoSeeds(const std::vector<edm::Ptr<l1t::HGCalCluster>>& clustersPtrs,
                                            std::vector<std::pair<GlobalPoint, double>>& seedPositionsEnergy) {
-  /* put clusters into an r/z x phi histogram */
-  Histogram histoCluster = fillHistoClusters(clustersPtrs);
-
-  Histogram smoothHistoCluster;
-  if (seedingSpace_ == RPhi) {
-    /* smoothen along the phi direction + normalize each bin to same area */
-    Histogram smoothPhiHistoCluster = fillSmoothPhiHistoClusters(histoCluster, binsSumsHisto_);
-
-    /* smoothen along the r/z direction */
-    smoothHistoCluster = fillSmoothRPhiHistoClusters(smoothPhiHistoCluster);
-  } else if (seedingSpace_ == XY) {
-    smoothHistoCluster = fillSmoothHistoClusters(histoCluster, smoothing_ecal_, Bin::Content::Ecal);
-    smoothHistoCluster = fillSmoothHistoClusters(smoothHistoCluster, smoothing_hcal_, Bin::Content::Hcal);
-    // Update sum with smoothen ECAL + HCAL
-    for (int z_side : {-1, 1}) {
-      for (unsigned x1 = 0; x1 < nBins1_; x1++) {
-        for (unsigned x2 = 0; x2 < nBins2_; x2++) {
-          auto& bin = smoothHistoCluster.at(z_side, x1, x2);
-          bin.values[Bin::Content::Sum] = bin.values[Bin::Content::Ecal] + bin.values[Bin::Content::Hcal];
-        }
+
+  for(std::vector<unsigned>::size_type it = 0; it != vnBins1_.size(); it++) {
```

Comment:
Shorter to use `size_t` instead of the lengthy `std::vector<unsigned>::size_type`